lib/public/views/DataPasses/ActiveColumns/dataPassesActiveColumns.js

```diff
@@ -40,7 +40,7 @@ export const dataPassesActiveColumns = {
             runsCount === 0
                 ? 'No runs'
                 : frontLink(
-                    h('.flex-row.g3', [h('.f6.badge.bg-gray-light.black', runsCount), 'Runs']),
+                    h('.flex-row.g3', h('.f6.badge.bg-gray-light.black', runsCount)),
```

Comment:
You can simply display them as normal links
(btw if you have only one child, you can remove the flex-row and gap)